sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -24,3 +24,17 @@ select left("abcd", 2), left("abcd", 5), left("abcd", '2'), left("abcd", null);
 select left(null, -2), left("abcd", -2), left("abcd", 0), left("abcd", 'a');
 select right("abcd", 2), right("abcd", 5), right("abcd", '2'), right("abcd", null);
 select right(null, -2), right("abcd", -2), right("abcd", 0), right("abcd", 'a');
+
+-- turn on concatBinaryAsString
+set spark.sql.function.concatBinaryAsString=false;
```

Comment:
turn on?

Reply:
Since most of other dbms-like systems concat binary inputs as binary, IMO turning off by default is okay to me.

Reply:
I meant you said turn on in the comment (L28).

Reply:
oh....